devicetypes/stelpro/stelpro-ki-zigbee-thermostat.src/stelpro-ki-zigbee-thermostat.groovy

```diff
@@ -309,8 +309,7 @@ def parse(String description) {
 
 				// If the user does not want to see the Idle and Heating events in the event history,
 				// don't show them. Otherwise, don't show them more frequently than 30 seconds.
```

Comment:
If we chose to remove this preference, this comment will need to be updated to reflect that. If we only change the preference text, then this comment can stay as-is.